include/rocksdb/block_cache_trace_writer.h

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2021, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+#pragma once
+
+#include "rocksdb/options.h"
+#include "rocksdb/system_clock.h"
+#include "rocksdb/table_reader_caller.h"
+#include "rocksdb/trace_reader_writer.h"
+#include "rocksdb/trace_record.h"
+
+namespace ROCKSDB_NAMESPACE {
+enum Boolean : char { kTrue = 1, kFalse = 0 };
```

Comment:
Good question. The original definition of BlockCacheTraceRecord in block_cache_tracer.h used it and I just moved the definitions. There is a case for using an enum instead of ```bool``` if the constants have descriptive names, as it improves readability. But in this case, the constants are just ```kTrue``` and ```kFalse```, which seems rather pointless. It might make more sense if we use something like ```enum CacheLookupResult : char { kCacheHit = 1, kCacheMiss = 0};```.